test/client.js

```diff
@@ -39,7 +39,7 @@ describe('http', () => {
       }
 
       fs.exists(filename, function (exists) {
-        if (exists) {
+        if (exists && filename[filename.length - 1] !== "/") {
```

Comment:
Single quote please. Also, what's the intention here? To check if filename is really a file? If so, we should probably use Node API to check.